app/utils/pgp.js

```diff
@@ -1,7 +1,9 @@
 const openpgp = require('openpgp')
 openpgp.initWorker({ path: '../node_modules/openpgp/dist/openpgp.worker.min.js' })
+/* eslint-disable */
```

Comment:
Can you just disable each of the lines individually instead of using this block disable?